src/view/items/element/specials/Select.js

```diff
@@ -65,8 +58,15 @@ export default class Select extends Element {
 			let optionWasSelected;
 
 			options.forEach( o => {
+				let shouldSelect = false;
 				const optionValue = o._ractive ? o._ractive.value : o.value;
-				const shouldSelect = isMultiple ? valueContains( selectValue, optionValue ) : selectValue == optionValue;
+				const isMultiple = this.getAttribute( 'multiple' );
+
+				if (isMultiple) {
+					shouldSelect = this.valueContains ( selectValue, optionValue );
+				} else {
+					shouldSelect = this.compare(optionValue, selectValue);
+				}
```

Comment:
```js
let shouldSelect = isMultiple ? this.valueContains( selectValue, optionValue )
                              : this.compare(optionValue, selectValue);
```

Also, the `let shouldSelect = false;` up top might be unnecessary. It's initial value will always be overridden.